toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -575,6 +575,9 @@ Specifies the configuration for dm-verity integrity verification.
 Note: Currently only root partition (`/`) is supported. Support for other partitions
 (e.g. `/usr`) may be added in the future.
 
+Note: The [filesystem](#filesystem-type) item pointing to this verity device, must
+include the `ro` option in the [mountPoint.options](#options-string).
```

Comment:
so are we allowing it or requiring it? also why is it required? (curious - don't know a lot about this topic)

Reply:
It's now required for verity. (Previously we would add the flag silently for verity.) And it is now an optional for non-verity.